### ISSUE TITLE
Pretrained models 

### ISSUE THREAD

**Issue by lastapple** [odegeasslbc/Self-Supervised-Sketch-to-Image-Synthesis-PyTorch#2]
Are pretrained models available?  Can you upload them to Google drive？Thanks!

**Comment by odegeasslbc** [comment]
Sorry, we can't release a pre-trained model due to commercial constraints. However, the model shall be trained with the provided code and data, a single GPU is able to reproduce similar results.

**Comment by lastapple** [comment]
Ok, thanks! I‘ll train the model.

**Comment by lyhh123** [comment]
Do all models need to be trained ? Including VGG?

**Comment by bingchenlll** [comment]
> Do all models need to be trained ? Including VGG?

No, if I understand your question correctly, VGG for the LPIPS loss does not need to be trained. The pre-trained model will be automatically downloaded and used.

**Comment by lyhh123** [comment]
> 
> 
> > Do all models need to be trained ? Including VGG?
> 
> No, if I understand your question correctly, VGG for the LPIPS loss does not need to be trained. The pre-trained model will be automatically downloaded and used.
> Thanks for your reply. The problem has been solved.

**Comment by lyhh123** [comment]
> 
> 
> > > Do all models need to be trained ? Including VGG?
> > 
> > 
> > No, if I understand your question correctly, VGG for the LPIPS loss does not need to be trained. The pre-trained model will be automatically downloaded and used.
> Thanks for your reply. The problem has been solved.

**Comment by ZhouCX117** [comment]
@lyhh123 @odegeasslbc Hi, how  to automatically download the vgg-feature-weights.pth used in sketch_styletransfer/train.py?
